Dolapo47/Auto-Mart
# Ch setup database #166808743

#### What does this PR do?
- A summary of what this task is about
This PR creates the database for the app
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
created an elephant sql account
connected elephantsql to github
wrote the db config file
and ran npm run db-migrate to create the tables
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
it can be tested by running npm 'run db-migrate' which creates the tables in the database

#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#166808743](https://www.pivotaltracker.com/n/projects/2346179)`)
